src/Umbraco.Web/Editors/PreviewController.cs

```diff
@@ -99,7 +99,7 @@ public JavaScriptResult Application()
         /// </summary>
         /// <returns></returns>
         [UmbracoAuthorize]
-        public ActionResult Frame(int id, string culture)
+        public ActionResult Frame(int id, string culture, string segment)
```

Comment:
This is a breaking change, we'll need to obsolete the old method signature and tell people to use the new one including the segment.

Reply:
Hi @nul800sebastiaan 

I don't think many people will be using this method directly in their code. And it is a route...so if the parameter is not present it will be empty by default.

So should I fix this ?

Dave

Reply:
Ah yes, I forgot how the routing for webapi works! Agreed that nobody should be relying on this in their custom c# code. All good then! 👍